go.mod

```diff
@@ -50,14 +50,17 @@ require (
 	github.com/yvasiyarov/gorelic v0.0.6 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
 	go.opencensus.io v0.22.2 // indirect
-	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
+	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad
 	golang.org/x/net v0.0.0-20201021035429-f5854403a974 // indirect
-	golang.org/x/sys v0.0.0-20201117170446-d9b008d0a637
+	golang.org/x/sys v0.0.0-20201201145000-ef89a241ccb3
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools/v3 v3.0.3
 	mvdan.cc/sh/v3 v3.2.1
-	rsc.io/letsencrypt v0.0.3 // indirect
 )
 
-replace golang.org/x/crypto => github.com/sylabs/golang-x-crypto v0.0.0-20181006204705-4bce89e8e9a9
+replace (
+	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
+	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
```

Comment:
@jdolitsky Wrt upgrading this go module, the need for these replacements is curious to me. 

Is there a reason why these changes (https://github.com/deislabs/oras/pull/209/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L16) needed to be made in the upstream module to move those dependencies to invalid versions?

Reply:
It was the result of running `go mod tidy` - if you look at go.sum you'll notice same versions pinned. If you're able to help stabilize these in oras, please submit a PR and we will re-release

Reply:
After doing some reading, it looks like using replace statements within the downstream module is the only way to handle cases like this, where the upstream is using replacements. 

Can you add a comment above the `github.com/docker/distribution` and `github.com/docker/docker` replacements that they are required for `github.com/deislabs/oras v0.9.0` so that their intention is clear in the future. Thanks!

Reply:
@ikaneshiro done!